sys/kern/proc.c

```diff
@@ -14,11 +14,20 @@
 #include <sys/sched.h>
 #include <sys/malloc.h>
 #include <sys/vfs.h>
+#include <sys/sysinit.h>
 #include <bitstring.h>
 
-#define NPROC 64 /* maximum number of processes */
+/* Allocate PIDs from a reasonable range, can be changed as needed. */
+#define PID_MAX 255
+#define NBUCKETS ((PID_MAX + 1) / 4)
+#define PIDHASH(pid) (pid % NBUCKETS)
```

Comment:
It's good idea to surround `pid` with parenthesis. Just in case someone passed `pid + 1` to this macro ;-)

Reply:
Done.